docs/application-connector/docs/03-07-api-registration.md

```diff
@@ -0,0 +1,21 @@
+---
+title: API registration in Application Registry
+type: Details
+---
+
+### Supported formats
+
+The Application Registry supports API Specification in the following formats:
+- OpenAPI 2.0
+- OData 2.0
```

Comment:
OData XML is supported in the following versions: 2.0,3.0,4.0. OData JSON is not supported by documentation subsystem and won't be displayed in console.

Reply:
Done!